OpenCL_C.txt

```diff
@@ -2386,14 +2627,16 @@ address space:
   * A pointer that points to the `constant` address space cannot be cast or
     implicitly converted to the generic address space.
 
-A few examples follow.
+Examples:
 
 This is the canonical example.
 In this example, function `foo` is declared with an argument that is a
-pointer with no address space qualifier.
+pointer with unnamed generic address space address space qualifier.
 
 [source,c]
 ----------
+#ifdef __opencl_c_generic_address_space
```

Comment:
I'd prefer to use the comment syntax used elsewhere, especially since these examples are also valid in OpenCL C 2.0 without the `__opencl_c_generic_address_space` feature macro:

```c
// Note: these examples assume OpenCL C 2.0 or the
// __opencl_c_generic_address_space feature macro.
```

Reply:
I have changed the example to use comments instead although we could of course add a language version macro. But I think the comment conveys the message directly!  So it is better.